pkg/apis/crd/v1alpha1/types.go

```diff
@@ -289,6 +292,17 @@ type Advertisements struct {
 	Egress *EgressAdvertisement `json:"egress,omitempty"`
 }
 
+type Confederation struct {
+	// Enabled indicates whether BGP confederation is enabled.
+	Enabled bool `json:"enabled,omitempty"`
+
+	// Identifier specifies the confederation's ASN.
+	Identifier int32 `json:"identifier,omitempty"`
+
+	// Peers lists the ASNs of other members in the confederation.
+	Peers []int32 `json:"peers,omitempty"`
```

Comment:
I think the term "members" is typically used for BGP confederation. Should we name this field `MemberASNs`?

Reply:
Make sense. `MemberASN` is the better choice.